lib/streamlit/elements/multiselect.py

```diff
@@ -213,6 +214,11 @@ def multiselect(
             An optional tuple of args to pass to the callback.
         kwargs : dict
             An optional dict of kwargs to pass to the callback.
+        max_selections : int
+            The max selections that can be selected at a time.
+            This argument can only be supplied by keyword.
+        placeholder : str
+            An optional string to display when option is not selected.
```

Comment:
nit: maybe "A string to display when no options are selected. Defaults to 'Choose an option'"?

Reply:
https://github.com/streamlit/streamlit/pull/6901/commits/d14d3757294766d9d7704c3a926d4c26966d0519
Thank you for the suggestion. It would be better to specify the default value.